src/pages-conditional/wallets/find-wallet.js

```diff
@@ -87,6 +86,14 @@ const Header = styled.header`
   }
 `
 
+const InfoBannerContainer = styled.div`
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    margin-left: 2rem;
+    margin-right: 2rem;
+    margin-top: -2rem;
```

Comment:
Why negative margin? I suspect there's a better way to solve this, e.g. by reducing margin-top of the element above.

Reply:
Changed margins of the container above.